torchrl/envs/common.py

```diff
@@ -552,10 +541,56 @@ def to(self, device: DEVICE_TYPING) -> _EnvClass:
         return self
 
 
-class _EnvWrapper(_EnvClass, metaclass=abc.ABCMeta):
+class EnvStateful(EnvBase):
+    """
+    Env class for stateful envs in TorchRL.
```

Comment:
(I might have done the same so it's also a comment to myself)
We should avoid saying "in torchrl" unless it truly contrasts with something else. It's kind of an empty statement.